README.md

```diff
@@ -50,8 +50,7 @@ docker_repositories()
 docker_pull(
   name = "java_base",
   registry = "gcr.io",
-  # TODO(mattmoor): Replace this.
-  repository = "google-appengine/java",
+  repository = "distroless/java",
   # 'tag' is also supported, but digest is encouraged for reproducibility.
   digest = "sha256:deadbeef",
```

Comment:
Want to use a real digest here?

Reply:
Can we leave a note as to the recommended way to obtain the sha256 of the pulled image?

Reply:
You should be able to put `gcr.io/foo/bar:blah` into your browser, and then clicking "Show Pull Command" will contain the digest.

Reply:
Thanks.  Attaching screenshot for someone in the future.  Might be nice to include this info in the README.

<img width="1435" alt="screen shot 2017-08-26 at 3 58 25 pm" src="https://user-images.githubusercontent.com/50580/29745434-922450cc-8a77-11e7-8b78-b6b5957bfd2f.png">